src/sources/datadog_agent/mod.rs

```diff
@@ -90,6 +94,7 @@ pub struct DatadogAgentConfig {
     /// For a source component named `agent`, the received logs, metrics, and traces can then be
     /// configured as input to other components by specifying `agent.logs`, `agent.metrics`, and
     /// `agent.traces`, respectively.
+    #[configurable(metadata(docs::advanced))]
     #[serde(default = "crate::serde::default_false")]
     multiple_outputs: bool,
```

Comment:
I thought it was a bad experience leaving this on a single output - is hiding that a good idea?

Reply:
The intent is to actually make this default to `true`... but I'm deferring any changes of configuration value _defaults_ to another PR in the future once each proposed change has been fully discussed.